tests/common.py

```diff
@@ -0,0 +1,66 @@
+"""Tests common stuff."""
+import struct
+
+import eblob_kit
+
+
+def generate_key(start=0):
+    """Generate valid key for testing."""
+    return ''.join([chr((start + x) % 256) for x in xrange(eblob_kit.DiskControl.key_size)])
+
+
+def make_header(data_size=0, disk_size=0, position=0, flags=eblob_kit.RecordFlags(0), key=''):
+    """Create DiskControl with default values.
+
+    Parameters equals to eblob_kit.DiskControl constructor.
+
+    """
+    return eblob_kit.DiskControl(key=key, data_size=data_size, position=position, disk_size=disk_size, flags=flags)
+
+
+def empty_header():
+    """Construct mepty header.
+
+    Usually such a header should be considered 'non valid', used as a stub for tesing
+    purposes.
+
+    :rtype eblob_kit.DiskControl
+    """
+    return make_header()
+
+
+def header_to_bytes(header):
+    """Construct bytes sequence DiskControl header representation.
+
+    :param eblob_kit.DiskControl header: disk control header to pack into created buffer.
+
+    :rtype: str
+    """
+    fmt = '<{}s4Q'.format(eblob_kit.DiskControl.key_size)
+    return struct.pack(fmt, header.key, header.flags.flags, header.data_size, header.disk_size, header.position)
+
+
+def make_blob_byte_squence(sizes=[], fill=0xf1):
```

Comment:
It is bad practice to use list or dict or any mutable value in a default value for an argument.